meta-cli/src/deploy/actors/task/command.rs

```diff
@@ -78,8 +79,8 @@ impl CommandContext {
                 .to_string();
             let command = if !argv_str_fmtd.eq(&argv_str) {
                 // custom
-                let argv = argv_str_fmtd.split(' ').collect::<Vec<_>>();
-                let mut command = Command::new(argv[0]);
+                let argv = shell_words::split(&argv_str_fmtd).unwrap();
+                let mut command = Command::new(argv[0].clone());
```

Comment:
`split` may fail as the env variable is provided by the user, so maybe you should add `context` to it before unwrap or  propagate the error if the function returns a result.